### DISCUSSION TITLE
v0.17.0

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#1223] [Releases]
## What's New
* [Feature] Discord Webhook Functionality by @jonezy35 in https://github.com/alexjustesen/speedtest-tracker/pull/1196

## What's Changed
* [Chore] Round order of magnitude at 9 hundredths instead of tenths by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1217
* Laravel 10.45.1 Shift by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1212
* [Bug] Added default download and upload value when assessing if a threshold was breached by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1218
* [Chore] Fixed location of setting migrations by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1219
* [Chore] Refactored discord webhooks to use WebhookCall by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1221
* Release v0.17.0 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1222

## New Contributors
* @jonezy35 made their first contribution in https://github.com/alexjustesen/speedtest-tracker/pull/1196

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.16.6...v0.17.0

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.17.0'>v0.17.0</a>.</em>

**Comment by seh2000** [comment]
Hi Alex,
Got error when I updated to v0.17.0
`admin@Pi-hole1:~/docker/speedtest-tracker $ sudo docker-compose up -d
Pulling speedtest-tracker (ghcr.io/alexjustesen/speedtest-tracker:v0.17.0)...
ERROR: manifest unknown
`

**Reply by alexjustesen** [reply]
I'll look into why it didn't build but you can get ahead of the upcoming deprecation by switching images: https://github.com/alexjustesen/speedtest-tracker/issues/1224

**Reply by seh2000** [reply]
Switched as per #1224
Now on v0.17.0

![image](https://github.com/alexjustesen/speedtest-tracker/assets/74658646/ce7b174a-75f0-48ae-a207-0372d70a0f24)

I noted Mbps is back in stead of Gbps.

**Comment by masterwishx** [comment]
latest docker image is 0.16.6 ! no image for 0.17.0 ...

**Reply by alexjustesen** [reply]
I'll look into why it didn't build but you can get ahead of the upcoming deprecation by switching images: https://github.com/alexjustesen/speedtest-tracker/issues/1224

**Comment by hansenc0705** [comment]
updated but notifications page is giving a 500 server error.

Do you have a discord?

**Reply by masterwishx** [reply]
yes

**Reply by masterwishx** [reply]
updated to 0.17.0 ? 

![image](https://github.com/alexjustesen/speedtest-tracker/assets/28630321/073ce7c7-eccd-46bf-934a-a36dc41da1c5)

**Reply by hansenc0705** [reply]
docker image - App: 0.17.0

**Reply by masterwishx** [reply]
What link ?

**Reply by alexjustesen** [reply]
I'll look into why it didn't build but you can get ahead of the upcoming deprecation by switching images: https://github.com/alexjustesen/speedtest-tracker/issues/1224

**Reply by hansenc0705** [reply]
ah will switch

its on ~/admin/notification-page

**Reply by alexjustesen** [reply]
@hansenc0705 if you're getting an error can you open an issue and please includes the logs.

**Reply by masterwishx** [reply]
linuxserver.io avalible but not here ...

**Reply by hansenc0705** [reply]
looks like im using the linux server.io image
![image](https://github.com/alexjustesen/speedtest-tracker/assets/10247644/78db5857-86aa-43a4-9bd8-fb3b5d15636c)

**Reply by masterwishx** [reply]
@alexjustesen  are you stopping to use you images , only lscr.io/linuxserver/speedtest-tracker will be avalible ?

**Reply by alexjustesen** [reply]
@masterwishx after the `0.17.x` release cycle yes. Getting Docker images off my plate gives me more time to work on the application 💪

**Comment by masterwishx** [comment]
@jonezy35 @alexjustesen Thanks a lot for this very needed function (Discord Webhook Functionality ) , that was waiting a long time :)

**Reply by jonezy35** [reply]
Happy to contribute :)
